library/ecp.c

```diff
@@ -1051,6 +1052,39 @@ static int ecp_modp( mbedtls_mpi *N, const mbedtls_ecp_group *grp )
     return( ret );
 }
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+
+int mbedtls_ecp_tls13_read_point( const mbedtls_ecp_group *grp,
```

Comment:
I do not think we need to add this TLS specific function in ecp.c. The code of this function can be put directly in  `ecdh_read_params_internal`.